proxy/connect/connect.go

```diff
@@ -161,6 +183,13 @@ sendLoop:
 		d.logger.WithError(err).Error("failed to close connection")
 	}
 
+	d.statsd.Count(
+		"veneur_proxy.forward.metrics_count", int64(len(d.sendChannel)),
+		[]string{"error:dropped"}, 1.0)
+	for range d.sendChannel {
```

Comment:
OOC: Why do we need to drain the channel? Hasn't the consumer been cancelled at this point, and channel closed above? It should be garbage collected along with the destination struct